pkg/yurthub/server/server.go

```diff
@@ -170,6 +171,10 @@ func registerHandlers(c *mux.Router, cfg *config.YurtHubConfiguration, certifica
 
 	// register handler for metrics
 	c.Handle("/metrics", promhttp.Handler())
+
+	// register handler for ota upgrade
+	c.Handle("/openyurt.io/v1/pods", ota.GetPods(clientset)).Methods("GET")
+	c.Handle("/openyurt.io/v1/namespaces/{ns}/pods/{podname}/update", ota.UpdatePod(clientset)).Methods("POST")
```

Comment:
according to https://github.com/openyurtio/openyurt/pull/921 proposal, the path should be the following:
```
/openyurt.io/v1/pods --> /pods
/openyurt.io/v1/namespaces/{ns}/pods/{podname}/update --> /openyurt.io/v1/namespaces/{ns}/pods/{podname}/upgrade
```